src/lib/common-yargs.js

```diff
@@ -0,0 +1,191 @@
+const fs = require('fs')
+const parseGitHubURL = require('./parse-github-url')
+
+const withToken = options => yargs =>
+	yargs
+		.option(
+			'token',
+			Object.assign(
+				{},
+				{
+					alias: 't',
+					describe: 'GitHub personal access token. \nGenerate one at https://github.com/settings/tokens.',
+					type: 'string',
+				},
+				options
+			)
+		)
+		.check(argv => {
+			if (!argv.token && !process.env.GITHUB_PERSONAL_ACCESS_TOKEN) {
+				throw new Error('Missing required argument: token.')
+			}
+			return true
+		})
+		.default('token', process.env.GITHUB_PERSONAL_ACCESS_TOKEN, 'process.env.GITHUB_PERSONAL_ACCESS_TOKEN')
```

Comment:
Users might not know what `process.env` is. `$THING` is well understood to be an environment variable.

```suggestion
		.default('token', process.env.GITHUB_PERSONAL_ACCESS_TOKEN, '$GITHUB_PERSONAL_ACCESS_TOKEN')
```